File: Angular2PoC/RoutingAndNavigation/app/log/day.model.ts
import {DateTimeModel} from './datetime.model';
import {EntryModel} from './entry.model';

export class DayModel {
    constructor(
        public day: DateTimeModel,
        public arrive: DateTimeModel,
        public leave: DateTimeModel,
        public inOffice: string,
        public outOfOffice: string,
        public logEntries: EntryModel[]) { }
}

File: OfficeHoursXamarin/OfficeHours/OfficeHours/Models/DayModel.cs
using System;
using System.Collections.Generic;

namespace OfficeHours
{
	public class DayModel
	{
		public DateTime Day { get; set; }
		public DateTime Arrive { get; set; }
		public DateTime Leave { get; set; }
		public TimeSpan InOffice { get; set; }
		public TimeSpan OutOfOffice { get; set; }

		public List<EntryModel> Entries { get; set; }

		public DayModel ()
		{
		}
	}
}

